From 1dd3c8f44b157911d8b3a9f0c7632c5c2eaec0ae Mon Sep 17 00:00:00 2001
From: Hardik T Shah <hardik.t.shah@intel.com>
Date: Sat, 23 Apr 2016 18:16:01 +0530
Subject: [PATCH 3897/4301] ASoC:SKL: Pass FPGA PDM Codec for CNL Platform.

commit c78c0f5a9db38a92344cf18cad956d0cf202fb8c from
git://git@github.com:01org/linux-intel-4.9.git

Pass the PDM Machine driver name for CNL Platform.
This patch is for testing only.

Change-Id: If72a4490bc9d5e074d6617080238288bc6268db4
Signed-off-by: Hardik T Shah <hardik.t.shah@intel.com>
Reviewed-on: https://git-gar-1.devtools.intel.com/gerrit/8988
---
 sound/soc/intel/skylake/skl.c |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/sound/soc/intel/skylake/skl.c b/sound/soc/intel/skylake/skl.c
index 92bfb02..d19ba35 100644
--- a/sound/soc/intel/skylake/skl.c
+++ b/sound/soc/intel/skylake/skl.c
@@ -439,7 +439,13 @@ static int skl_free(struct hdac_ext_bus *ebus)
 static struct sst_acpi_mach sst_cnl_devdata[] = {
 	{
 		.id = "dummy",
+#ifdef CONFIG_SND_SOC_INTEL_CNL_WM8281_MACH
 		.drv_name = "cnl_florida",
+#elif CONFIG_SND_SOC_SVFPGA
+		.drv_name = "cnl_svfpga",
+#elif CONFIG_SND_SOC_INTEL_CNL_CS42L42_MACH
+		.drv_name = "cnl_cs42l42",
+#endif
 		.fw_filename = "intel/dsp_fw_cnl.bin",
 	},
 };
-- 
1.7.5.4

